platform/ios/os_ios.mm

```diff
@@ -571,9 +571,14 @@ void register_dynamic_symbol(char *name, void *address) {
 	return ret;
 }
 
-void OS_IOS::vibrate_handheld(int p_duration_ms) {
+void OS_IOS::vibrate_handheld(int p_duration_ms, float p_amplitude) {
 	if (ios->supports_haptic_engine()) {
-		ios->vibrate_haptic_engine((float)p_duration_ms / 1000.f);
+		if (p_amplitude == -1.0) {
+			p_amplitude = 1.0;
+		}
```

Comment:
I tried to check the iOS docs but they're as usual surprisingly unhelpful to document default values: https://developer.apple.com/documentation/corehaptics/chhapticevent/parameterid/hapticintensity?language=objc

Did you confirm that 1.0 is the default value for `HapticIntensity` when not set explicitly?

Reply:
I didnt since i couldnt find thr docs as well and i dont have an apple device to test on.

Reply:
CC @bruvzg

Reply:
Should i do a workaround that simply doesnt give that argument?
(I'll do so when im home)

Reply:
It is done.